bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -1,32 +1,28 @@
 #GoAndKickBallMapGoal
 $BallKickArea
-    LEFT --> @Stop, @KickBall + foot:left
-    RIGHT --> @Stop, @KickBall + foot:right
+    LEFT --> @Stop, @KickBallDynamic
+    RIGHT --> @Stop, @KickBallDynamic
     FAR --> @GoToBall + target:map_goal
 
 #GoAndKickBallDetectionGoal
 $BallKickArea
-    LEFT --> @Stop, @KickBall + foot:left
-    RIGHT --> @Stop, @KickBall + foot:right
+    LEFT --> @Stop, @KickBallDynamic
+    RIGHT --> @Stop, @KickBallDynamic
     FAR --> @GoToBall + target:detection_goal
 
 #GoAndKickBallAway
 $BallKickArea
-    LEFT --> @Stop, @KickBall + foot:left
-    RIGHT --> @Stop, @KickBall + foot:right
+    LEFT --> @Stop, @KickBallDynamic
+    RIGHT --> @Stop, @KickBallDynamic
     FAR --> @GoToBall + target:none
 
 #NoLocalizationPlayerBehavior
 $BallSeen
-    YES --> $BallDangerous
-        YES --> $BallClose
-            YES --> $GoalSeen
-                YES --> #GoAndKickBallDetectionGoal
-                NO --> @GoToBall + target:none, @SearchGoal
-            NO --> @GoToBall + target:none
-        NO --> $BallInDefensiveArea
-            YES --> @GoToBlockPosition, @LookAtBall
-            NO --> @LookAtBall
+    YES --> $BallClose
+        YES --> $GoalSeen
+            YES --> #GoAndKickBallDetectionGoal
+            NO --> @GoToBall + target:none, @SearchGoal
+        NO --> @GoToBall + target:none
```

Comment:
this could end up in a merge conflict. You should use everything from master there